src/components/profile/ProfileDataTable.js

```diff
@@ -54,15 +54,15 @@ const ProfileDataTable = ({
 
   // phone handlers
   const onPhoneInputFocus = useCallback(() => setLockSubmit(true), [setLockSubmit])
-  const onPhoneInputChange = useCallback(value => onChange({ ...profile, mobile: value }), [onChange, profile])
+  const onPhoneInputChange = useCallback(value => onChange({ ...profile, mobile: value || '' }), [onChange, profile])
```

Comment:
not needed, it will be empty value or null if input field is empty

Reply:
Yes, we getting undefined, if input field is empty, but if we send `mobile: undefined` to server while updating profile - server doesn't save the empty mobile (no error, just stayed the same). So, I just setting `''` value if mobile is empty and then checking it while updating. I must change it back according to your suggestion or leave my variant?